rubocop-govuk.gemspec

```diff
@@ -11,11 +11,13 @@ Gem::Specification.new do |spec|
 
   spec.files         = Dir["config/**/*", "*.md"]
 
+  spec.required_ruby_version = ">= 2.7"
+
   spec.add_development_dependency "rake", "~> 13"
 
-  spec.add_dependency "rubocop", "0.87.1"
-  spec.add_dependency "rubocop-ast", "0.8.0"
-  spec.add_dependency "rubocop-rails", "2.8.1"
+  spec.add_dependency "rubocop", "~> 1.7.0"
+  spec.add_dependency "rubocop-ast", "~> 1.4.0"
+  spec.add_dependency "rubocop-rails", "~> 2.9.1"
   spec.add_dependency "rubocop-rake", "0.5.1"
-  spec.add_dependency "rubocop-rspec", "1.42.0"
+  spec.add_dependency "rubocop-rspec", "~> 2.1.0"
```

Comment:
Agree, I can't imagine it would be too much of an issue. Rubocop versioning has this to say since their move past 1.0:

> RuboCop is stable between major versions, both in terms of API and cop configuration. We aim the ease the maintenance of RuboCop extensions (by keeping the API stable) and the upgrades between RuboCop releases (by not enabling new cops and changing the configuration of existing cops). All big (breaking) changes are reserved for major releases.

And:

> Bug-fix (point) releases (if any) address only serious bugs and never contain new features.

We already have `NewCops: disable` in the default config, so any new rules between explicit releases of `rubocop-govuk` should not impact apps when patch level releases come out.

Reply:
Great stuff 👍  Yeah my concern lies from a recent memory that the version of [rubocop-ast could cause a bit of a pain](https://github.com/alphagov/rubocop-govuk/pull/117) hopefully they're release process will mean that no longer happens at patch versions.